en/appendices/5-3-migration-guide.rst

```diff
@@ -54,3 +54,7 @@ Validation
 
 - ``ipOrRange()`` validation has has been added to check for an IP or a range (subnet).
 
+TestSuite
+---------
+
+- `assertRedirectBack()` added to assert a successful redirect has been made to the same previous URL.
```

Comment:
Invalid code fencing will be fixed with https://github.com/cakephp/docs/pull/8031 merge.